pyomo/contrib/doe/tests/test_doe_solve.py

```diff
@@ -443,5 +445,102 @@ def test_reactor_grid_search_bad_model(self):
         )
 
 
```

Comment:
```suggestion

@unittest.skipIf(not ipopt_available, "The 'ipopt' solver is not available")
```
@smondal13 I think you just need to skip these tests when ipopt is not available to get the last check passing.